packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -449,47 +474,68 @@ const DataSeriesFormItemModal = ({
               />
             </div>
           )}
-        </div>
 
-        {(type === CARD_TYPES.VALUE || type === CARD_TYPES.IMAGE) && (
-          <div className={`${baseClassName}--input-group`}>
+          {hasTooltip && (
             <div className={`${baseClassName}--input-group--item`}>
               <TextInput
-                id={`${id}_attribute-unit`}
-                labelText={mergedI18n.dataItemEditorDataItemUnit}
+                id={`${id}_attribute-tooltip`}
+                labelText={mergedI18n.dataItemEditorDataItemTooltip}
                 light
-                placeholder={`${mergedI18n.example}: %`}
                 onChange={(evt) =>
                   setEditDataItem({
                     ...editDataItem,
-                    unit: evt.target.value,
+                    tooltip: evt.target.value,
                   })
                 }
-                value={editDataItem.unit}
+                value={editDataItem.tooltip}
```

Comment:
I think that changing expected property names could cause backward compatibility issues. If you need to change it, maybe use a fallback logic to also accept the previous one.